### ISSUE TITLE
[Legitimate Site Blocked] shibasocialclub.com

### ISSUE THREAD

**Issue by komninoschat** [MetaMask/eth-phishing-detect#6337]
https://shibasocialclub.com/

I am the developer of this NFT drop that sold out for $8m+.

Here is the OS collection that links back to the website to confirm authenticity:
http://opensea.io/collection/shiba-social-club

Kindly remove the phishing warning.

Thank you.

**Comment by devenblake** [comment]
It doesn't look like this site should be blocked. Can you screenshot the phishing warning and include the URL to which the MetaMask extension redirects you?

**Comment by komninoschat** [comment]
Hi @devenblake,

Thanks for the swift response. Here's the info requested:
![Komninos Screenshot - 3EiOz3iz](https://user-images.githubusercontent.com/29049104/148397159-9b45da1a-4b2d-45bc-a00a-c204fb9ae893.png)
`chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn/phishing.html#hostname=shibasocialclub.com&href=https%3A%2F%2Fshibasocialclub.com%2F`

And here is a video of me navigating to the site and getting blocked: https://share.komninos.dev/spzYVU

**Comment by devenblake** [comment]
I'm not sure what's happening with this. It's not in the blocklist but I don't see how it'd be auto-flagged.

**Comment by devenblake** [comment]
Looks like it wasn't on the blocklist on *my local branch* because you filed this issue so soon after it went in - this was explicitly blocked with #6332. Sorry about that.

**Comment by komninoschat** [comment]
Thanks for the effort @devenblake - and great that you found the cause of the issue.

Do you have an ETA on getting unblocked?

**Comment by devenblake** [comment]
You link to the OpenSea collection http://opensea.io/collection/shiba-social-club but I see that that collection links to `shiba-social-club.com`, not `shibasocialclub.com`. The linked Discord also says the only official web link for Shiba Social Club is `shiba-social-club.com`.

**Comment by komninoschat** [comment]
@devenblake we had to migrate to the `shiba-social-club.com` domain to avoid the Metamask popup. If you search the old domain in the Discord server, you'll see that it was the one used previously.

Also, both domains point to the same Google Cloud load balancer

**Comment by devenblake** [comment]
@komninoschat looks like it was an accident that blocked it, should be all set now. Feel free to re-open or file a new issue if the site's still blocked (though maybe wait a day or two for the caches to clear).
